src/controllers/submitUrlController.js

```diff
@@ -68,8 +73,12 @@ class SubmitUrlController extends UploadController {
     try {
       return await axios.head(url)
     } catch (err) {
-      console.error(err)
-      return null
+      if (['ENOTFOUND', 'ECONNREFUSED'].includes(err.code)) {
```

Comment:
I think leaving the return value as `Promise<Response?>` (or `Promise<Response | null>`) makes more sense than  `Promise<Response | false>`.

Reply:
changed to return null instead of false